src/resources/elements/primeDesignSystem/pform-input/pform-input.html

```diff
@@ -26,7 +26,7 @@
   <!-- Validation message section -->
   <p
     class="feedbackMessage ${ validationStateExists(validationState) ? 'feedbackMessageVisible': '' } ${validationState + 'Message'}">
-    ${validationMessage}
+    ${!inputReference.disabled ? validationMessage : ''}
     <slot name="validation"></slot>
   </p>
```

Comment:
Does it make sense to `if.bind` the `<p>` element here?
If I understand correctly, in the `disabled` case, we don't show any validation message..
..except for when `slot="validation"` is used?

I guess, I'm just a bit confused about the intended behavior of `pform-input` in case of
- validation message and
- disabled

Usage of `slot="validation"` example: (in the demo page)

```html
  <pform-input label="Name" validation-state.bind="field3">
    <pinput-text value.bind="field3"></pinput-text>

    <div slot="validation">       // <--- HERE
      <b style="color: orange"><i class="far fa-trash-alt"></i> This is my Custom validation </b> message
    </div>
  </pform-input
```

Reply:
There is an issue in Aurelia where the slot is not replaced by our custom HTML if the parent of the slot has an if statement. I couldn't make it work. In case of custom error messages, you will have to handle their visibility manually. We can have a call about it in case you know a solution to this problem. If so, then yes, the if.bind should be on the `p` tag.